requirements/udata.in

```diff
@@ -1 +1 @@
-udata@git+https://github.com/opendatateam/udata.git
+udata@git+https://github.com/maudetes/udata.git@feat/add-roles-in-harvest-extras
```

Comment:
Don't forget to rollback this change

Reply:
Done in https://github.com/datagouv/udata-front/pull/528/commits/a7318f6a84a050d253b279c87dc71073cdfc8fdd